libvirt.go

```diff
@@ -241,6 +241,17 @@ func (l *Libvirt) Disconnected() <-chan struct{} {
 	return l.disconnected
 }
 
+// IsDisconnected indicates whether or not there is still a connection to
```

Comment:
```suggestion
// IsDisconnected indicates whether or not there is currently a connection to
```

I'm hoping this will not explode if called before the first connection.